nerfstudio/scripts/train.py

```diff
@@ -95,6 +95,8 @@ def train_loop(local_rank: int, world_size: int, config: TrainerConfig, global_r
         config: config file specifying training regimen
     """
     _set_random_seed(config.machine.seed + global_rank)
+    if torch.cuda.is_available():
+        torch.cuda.set_device(local_rank)
```

Comment:
@jkulhanek is this necessary? if so I added the `if` statement to prevent the CI from complaining

Reply:
`torch.cuda.set_device()` is actually new to me, so I was just looking it up in the [PyTorch docs](https://pytorch.org/docs/stable/generated/torch.cuda.set_device.html) and saw:
> Usage of this function is discouraged in favor of [device](https://pytorch.org/docs/stable/generated/torch.cuda.device.html#torch.cuda.device). In most cases it’s better to use CUDA_VISIBLE_DEVICES environmental variable.

If you think it's a good change despite being unrelated to this PR I'm fine with leaving it, I just wasn't sure why it was needed, if it breaks things for people using `CUDA_VISIBLE_DEVICES`, etc.

Reply:
I removed set_device as it is unrelated. We can add it once the need arises I guess.

Reply:
Thanks! In that case the PR looks good to me.